From 017322ee28c1579ce6c81904842aaada9c82403c Mon Sep 17 00:00:00 2001
From: Ting Liu <b28495@freescale.com>
Date: Wed, 19 Dec 2012 04:39:57 -0600
Subject: [PATCH 10/27] eglibc: run libm-err-tab.pl with specific dirs in ${S}

libm-err-tab.pl will parse all the files named "libm-test-ulps"
in the given dir recursively. To avoid parsing the one in
${S}/.pc/ (it does exist after eglibc adds aarch64 support,
${S}/.pc/aarch64-0001-glibc-fsf-v1-eaf6f205.patch/ports/sysdeps/
aarch64/libm-test-ulps), run libm-err-tab.pl with specific dirs
in ${S}.

Upstream-Status: inappropriate [OE specific]

Signed-off-by: Ting Liu <b28495@freescale.com>
---
 manual/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/manual/Makefile b/manual/Makefile
index 5382208..6b701b0 100644
--- a/manual/Makefile
+++ b/manual/Makefile
@@ -105,7 +105,8 @@ $(objpfx)libm-err.texi: $(objpfx)stamp-libm-err
 $(objpfx)stamp-libm-err: libm-err-tab.pl $(wildcard $(foreach dir,$(sysdirs),\
 						     $(dir)/libm-test-ulps))
 	pwd=`pwd`; \
-	$(PERL) $< $$pwd/.. > $(objpfx)libm-err-tmp
+	$(PERL) $< $$pwd/../ports > $(objpfx)libm-err-tmp
+	$(PERL) $< $$pwd/../sysdeps >> $(objpfx)libm-err-tmp
 	$(move-if-change) $(objpfx)libm-err-tmp $(objpfx)libm-err.texi
 	touch $@
 
-- 
2.1.4

